superset-frontend/cypress-base/package.json

```diff
@@ -5,19 +5,19 @@
   "scripts": {
     "cypress": "cypress",
     "cypress-run-chrome": "cypress run --browser chrome --headless",
-    "cypress-debug": "cypress open --config watchForFileChanges=true"
+    "cypress-open-debug": "cypress open --config watchForFileChanges=true"
```

Comment:
Just a nit. I think many people might be used to this command and changing it might cause a surprise. Not too sure I see the benefit compared to just leaving it as it was.